packages/gestalt-codemods/46.0.0/new_badge_props.js

```diff
@@ -0,0 +1,95 @@
+/*
+ * Converts
+ *  <Module badgeText="new" /> to <Module badge={{text: "New"}} />
+ *  <Dropdown.Item || .Link badgeText="new" {...} /> to <Dropdown.Item || .Link badge={{text: "New"}} {...} />
+ */
+
+// yarn codemod --parser=flow -t=packages/gestalt-codemods/46.0.0/new_badge_props.js relative/path/to/your/code
+
+// Run "yarn codemod detectManualReplacement ~/path/to/your/code --component=Module --subcomponent=Expandable --prop=badgeText" to find locations of badgeText inside Module.Expandable
+
+export default function transformer(file, api) {
+  const j = api.jscodeshift;
+  const src = j(file.source);
+  let localIdentifierNames;
+  let fileHasModifications = false;
+
+  src.find(j.ImportDeclaration).forEach((path) => {
+    const decl = path.node;
+    // Not Gestalt, bail
+    if (decl.source.value !== 'gestalt') {
+      return null;
+    }
+
+    // Find the local names of Dropdown and Module imports
+    localIdentifierNames = decl.specifiers
+      .filter((node) => ['Dropdown', 'Module'].includes(node.imported.name))
+      .map((node) => node.local.name);
+    return null;
+  });
+
+  // No Module or Dropdown imports, bail
+  if (!localIdentifierNames) {
```

Comment:
Given the `.filter` and `.map` calls, I believe `localIdentifierNames` will always at least be an empty array, which is truthy, so this check will never bail early. We need to check for array length instead